<launch>
  <!-- VRPN Server -->
  <arg name="server" default="mocap.local"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883
      update_frequency: 100.0
      frame_id: world
      use_server_time: false
      broadcast_tf: true
      refresh_tracker_frequency: 1.0
    </rosparam>

    <!-- Drone1 Remap -->
    <remap from="/vrpn_client_node/drone1/pose" to="drone1/mavros/vision_pose/pose" />

    <!-- Drone2 Remap -->
    <remap from="/vrpn_client_node/drone2/pose" to="drone2/mavros/vision_pose/pose" />

    <!-- Drone3 Remap -->
    <remap from="/vrpn_client_node/drone3/pose" to="drone3/mavros/vision_pose/pose" />
    </node>
</launch>
